osv2/mget.go

```diff
@@ -78,45 +83,47 @@ func (m *MGetRequest) Do(ctx context.Context, client *opensearch.Client) (*MGetR
 		return nil, rErr
 	}
 
-	resp := &MGetResponse{
-		StatusCode: osResp.StatusCode,
```

Comment:
we can also remove these fields from the `MGetResponse` struct definition as well.

Reply:
Nice, I missed that. Thanks